src/Shaders/background.vertex.fx

```diff
@@ -78,6 +80,16 @@ void main(void) {
 
 #if defined(REFLECTIONMAP_EQUIRECTANGULAR_FIXED) || defined(REFLECTIONMAP_MIRROREDEQUIRECTANGULAR_FIXED)
     vDirectionW = normalize(vec3(finalWorld * vec4(position, 0.0)));
+
+    if (fFovMultiplier != 1.0) {

```

Comment:
TO optimize it we would like to add a #define instead of a "if"

Reply:
Branching on a uniform is much cheaper than on other data types, as the shader knows it's locked into a single value for its entire run. Further, it's in the vertex shader so it is not running that often. Given the power of modern GPUS and these mitigating factors, I thought it was worth having as a uniform instead of a define. As a uniform it can easily and cheaply be animated, like adjusting a pair of binoculars or smoothing the change in a video dome. Otherwise as a define it is forcing a shader recompile with every change to the value.

Reply:
I meant only replacing the if by a define
Everything else stays and you still have an uniform for the animations